docs/configure-archive-logs.md

```diff
@@ -59,3 +60,14 @@ spec:
     archiveLocation:
       archiveLogs: true
 ```
+
+## Why doesn't Argo Workflows recommend using this feature?
+
+Argo Workflows log storage facilities are quite basic. It is recommended that you use a combination of:
+
+* A fully featured kubernetes capable logging facility which will provide you with facilities for indexing, searching and managing of log storage.
```

Comment:
Caps K on 'kubernetes'.

Should possibly be `Kubernetes-capable`

Reply:
I'm not fighting .spelling over that, so just plain "Kubernetes capable".

Reply:
Switched to "Kubernetes-aware" per [below comments](https://github.com/argoproj/argo-workflows/pull/12597#discussion_r1473473938)